api/nodes/serializers.py

```diff
@@ -130,15 +130,15 @@ def update(self, instance, validated_data):
         return instance
 
 
-class NodePointersSerializer(JSONAPISerializer):
+class NodeLinksSerializer(JSONAPISerializer):
```

Comment:
Strictly speaking, these should probably be NodeNodeLinks, but that sounds dumb. Let's try it this way for the moment.